Automattic/wp-calypso
# Stats: Update shortcut styles for date picker

Related to #81939.

## Proposed Changes

Adds classes and basic styling for the shortcuts list.

<img width="396" alt="SCR-20230928-mdtd" src="https://github.com/Automattic/wp-calypso/assets/40267301/07905c22-51ad-421b-955d-4fd9f71f733e">

## Testing Instructions



1. Open Live branch.
2. Navigate to **Stats → Traffic**.
3. Enable feature flag: `?flags=stats/date-control`
4. Click on the green date picker control.
5. Confirm the shortcuts look like the screenshot above.
6. Confirm hover styling works.
7. Adding the `is-selected` class to any of the `li` tags via the Inspector should update the color and display the checkmark.

**Note:** Does not track selection in the control yet. That will be coming in a follow-up PR. We only provides the basic styling here.

## Pre-merge Checklist
